distributed/scheduler.py

```diff
@@ -1855,37 +1865,21 @@ def _transition(
 
                 recommendations.update(a_recs)
                 for c, new_msgs in a_cmsgs.items():
-                    msgs = client_msgs.get(c)
-                    if msgs is not None:
-                        msgs.extend(new_msgs)
-                    else:
-                        client_msgs[c] = new_msgs
+                    client_msgs.setdefault(c, []).extend(new_msgs)
                 for w, new_msgs in a_wmsgs.items():
-                    msgs = worker_msgs.get(w)
-                    if msgs is not None:
-                        msgs.extend(new_msgs)
-                    else:
-                        worker_msgs[w] = new_msgs
+                    worker_msgs.setdefault(w, []).extend(new_msgs)
 
                 recommendations.update(b_recs)
                 for c, new_msgs in b_cmsgs.items():
-                    msgs = client_msgs.get(c)
-                    if msgs is not None:
-                        msgs.extend(new_msgs)
-                    else:
-                        client_msgs[c] = new_msgs
+                    client_msgs.setdefault(c, []).extend(new_msgs)
                 for w, new_msgs in b_wmsgs.items():
-                    msgs = worker_msgs.get(w)
-                    if msgs is not None:
-                        msgs.extend(new_msgs)
-                    else:
-                        worker_msgs[w] = new_msgs
+                    worker_msgs.setdefault(w, []).extend(new_msgs)
```

Comment:
This codecov failure is worrying. Not a regression in this PR though. CC @fjetter

Reply:
Im aware but I'm not necessarily concerned. I wouldn't be surprised that this is just cargo cult.